### PR TITLE
[path] manual progression allowed

### PR DESCRIPTION
**Pull request by nlko**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Fixed tickets | 
Added option to path settings to allow/forbid users to manually set/change his/her progression.

### CODE REVIEW

File: plugin/path/Form/AbstractPathType.php [hunk 1]
```diff
@@ -18,6 +18,7 @@ public function buildForm(FormBuilderInterface $builder, array $options = [])
         $builder->add('breadcrumbs',                'checkbox', ['required' => false]);
         $builder->add('summaryDisplayed',           'checkbox', ['required' => false]);
         $builder->add('completeBlockingCondition',  'checkbox', ['required' => false]);
+        $builder->add('manualProgressionAllowed',   'checkbox', ['required' => true]);
```
Comment (Elorfin) [inline plugin/path/Form/AbstractPathType.php]: The field should not be required.

**Comment by Elorfin** [comment]
LGTM

File: plugin/path/Form/AbstractPathType.php (hunk 1 above)
Comment (nlko) [inline plugin/path/Form/AbstractPathType.php]: done

**Comment by LaurentGruber** [comment]
LGTM

**Comment by LaurentGruber** [comment]
LGTM

**Pull request merged**